index.html

```diff
@@ -4,7 +4,7 @@
 	<meta charset="utf-8" />
 	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 
-	<title>AppImage | Linux apps that run anywhere</title>
+	<title>AppImage | Linux applications that universally run anywhere</title>
```

Comment:
What is wrong with "apps"? 
What does "universally" mean?

Reply:
* I moved apps to the description as we have more room here and less room there
* Well if snap can claim universal, appimage is even more universal.